kubernetes/resource_kubernetes_api_service.go

```diff
@@ -33,9 +33,10 @@ func resourceKubernetesAPIService() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"ca_bundle": {
-							Type:        schema.TypeString,
-							Description: "CABundle is a PEM encoded CA bundle which will be used to validate an API server's serving certificate. If unspecified, system trust roots on the apiserver are used.",
-							Optional:    true,
+							Type:         schema.TypeString,
+							Description:  "CABundle is a PEM encoded CA bundle which will be used to validate an API server's serving certificate. If unspecified, system trust roots on the apiserver are used.",
+							Optional:     true,
+							ValidateFunc: validateBase64Encoded,
```

Comment:
That's not explicitly stated in the documentation but this PEM encoded CA bundle has to be base64 encoded too.

https://github.com/kubernetes/kubernetes/commit/b53b7f20622f1a6fa4d9b47e51860473a8683100#diff-b9357530db540633d17b5a068bbbfd2fR53